packages/dynamite/lib/src/openapi_builder.dart

```diff
@@ -1016,6 +1019,12 @@ class OpenAPIBuilder implements Builder {
 
       if (state.registeredJsonObjects.isNotEmpty) {
         output.addAll([
+          '@SerializersFor(const [',
+          for (final name in state.registeredJsonObjects) ...[
+            '$name,',
+          ],
+          '])',
+          r'final Serializers serializers = (_$serializers.toBuilder()..addPlugin(StandardJsonPlugin())).build();',
```

Comment:
Should the `_serializers` below removed after this change?

Reply:
I have it in a separate commit as theoretically the normal objects aren't yet migrated to it.

Reply:
Could you already do the migration in this PR or is it too big?

Reply:
I can sqash them if you want github.com/provokateurin/nextcloud-neon/pull/194/commits/b1c4273981ea9fc8e168ac78f04afeb279de9239

Reply:
Yes please, the commit looks good.